.buildkite/pipelines/code_coverage/daily.yml

```diff
@@ -0,0 +1,41 @@
+env:
+  SLACK_NOTIFICATIONS_CHANNEL: "#kibana-qa"
+  SLACK_NOTIFICATIONS_ENABLED: "true"
+  CODE_COVERAGE: 1
+notify:
```

Comment:
The `SLACK_NOTIFICATIONS_ENABLED` env vars above are for our custom Slack notifier, but this `notify:` block is for the one that's built-in to Buildkite, so you should probably pick one and remove the other.

Reply:
Which is best?